index.ts

```diff
@@ -0,0 +1,13 @@
+import type { Options, Validation } from './dist/types';
+import validate from './dist/validation';
+import workflowEventSchema from './dist/workflow-event-schema.json';
+import workflowStepTemplateSchema from './dist/workflow-step-template-schema.json';
+import workflowTemplateSchema from './dist/workflow-template-schema.json';
+
+export const schemas = {
+  workflowTemplateSchema,
+  workflowStepTemplateSchema,
+  workflowEventSchema,
+};
+
+export { Options, Validation, validate };
```

Comment:
move this file to `src/index.ts`, so that it get compiled into `dist/index.js` and `dist/index.d.ts`..